ts/WoltLabSuite/Core/Ui/Poll/Manager/Poll.ts

```diff
@@ -0,0 +1,205 @@
+/**
+ * Handles the poll UI.
+ *
+ * @author  Joshua Ruesweg
+ * @copyright  2001-2022 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Ui/Poll/Manager/Poll
```

Comment:
Why isn't this just `WoltLabSuite/Core/Ui/Poll.ts`? What is the point of nesting this into `/Manager/`, it doesn't add any value.